### DISCUSSION TITLE
求3dmigoto能用的好看的藿藿mod谢谢！（不要18+）（不要thicc)

### DISCUSSION THREAD

**Discussion by BlessedinHighlight** [share121/inter-knot#1802] [其他]
![Screenshot 2024-08-06 180807](https://github.com/user-attachments/assets/7b2b04b3-c1cf-4444-b95f-59758c633a74)

**Comment by hanyaser** [comment]
加mod游戏不会显示什么错误吗，想试试

**Reply by BlessedinHighlight** [reply]
看你是什么服务器，外服加mod基本上没限制也没人管，YouTube上就能搜到教程。国服就麻烦很多
